loadtest/control/actions.go

```diff
@@ -119,6 +124,7 @@ func JoinChannel(u user.User) UserActionResponse {
 	return UserActionResponse{Info: "no channel to join"}
 }
 
+// LeaveChannel removes the user from a random channel.
```

Comment:
Same as above. We leave the first channel we find in the store that the user is a member of.